generator/internal/parser/protobuf_test.go

```diff
@@ -1025,7 +1025,7 @@ func newTestCodeGeneratorRequest(t *testing.T, filename string) *pluginpb.CodeGe
 	return request
 }
 
-func checkMessage(t *testing.T, got genclient.Message, want genclient.Message) {
+func checkProtobufMessage(t *testing.T, got genclient.Message, want genclient.Message) {
```

Comment:
I am sure you noticed, but I think these helpers and the OpenAPI counterparts could be merged now.

Reply:
Ah I missed this! Will merge in a follow up.